Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/features/settings/model/view/IntSliderSetting.java

```diff
@@ -13,6 +13,11 @@ public IntSliderSetting(String file, String section, String key, int titleId, in
     mDefaultValue = defaultValue;
   }
 
+  public int getDefaultValue()
```

Comment:
Since this method signature is identical to the one in `PercentSliderSetting`, I think it would make sense to add this as an abstract method in `SliderSetting`. That way you don't have do add any casts in this PR.